### PR TITLE
Changing Array.blit to be just an alias for Array.Copy

### PR DESCRIPTION
**Pull request by forki**

### CODE REVIEW

File: src/fsharp/FSharp.Core.Unittests/FSharp.Core/Microsoft.FSharp.Collections/ArrayProperties.fs [hunk 1]
```diff
@@ -15,6 +16,16 @@ let distinctByStable<'a when 'a : comparison> (xs : 'a []) =
     isStable sorted
     
 [<Test>]
-let ``Seq.distinctBy is stable`` () =
+let ``Array.distinctBy is stable`` () =
```
Comment (forki) [inline src/fsharp/FSharp.Core.Unittests/FSharp.Core/Microsoft.FSharp.Collections/ArrayProperties.fs]: this is just a typo fix

**Comment by forki** [comment]
Ok I think this is done and I hereby propose this is a replacement for #1335

**Comment by dsyme** [comment]
@forki Please change the title, thanks

**Comment by dsyme** [comment]
Thanks for the contribution @forki , great work.

**Pull request merged**